CHANGELOG.md

```diff
@@ -6,6 +6,7 @@ Inspired from [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 - Added PHP 8.2 to the CI
 - Github workflow for changelog verification ([#92](https://github.com/opensearch-project/opensearch-php/pull/92))
 - Added class docs generator ([#96](https://github.com/opensearch-project/opensearch-php/pull/96))
+- Added optional `includePortInHostHeader` option to `ClientBuilder::fromConfig` ([#118](https://github.com/opensearch-project/opensearch-php/pull/118))
```

Comment:
nit: optional option is a bit strange, woudl just say "added includePortInHostHeader option to .."